src/2981R0_improving_our_safety_with_a_physical_quantities_and_units_library.md

```diff
@@ -1024,7 +1029,15 @@ static_assert(5 * h / (120 * min) == 0 * one);
 
 This is why floating point representation types are recommended as a default to store the numerical
 value of a quantity. Some popular physical units libraries even
-[forbid integer division at all](https://aurora-opensource.github.io/au/main/troubleshooting/#integer-division-forbidden).
+[forbid integer division at all](https://aurora-opensource.github.io/au/main/troubleshooting/#integer-division-forbidden). 
+
+The problem is similar to the one described in the section about accidental truncation of values 
+through conversion. While the consequent use of floating-point representation types may be a good idea,
+it is not always possible. Especially in close-to-the-metal applications and small embedded systems, the 
+use floating-point types is sometimes not an option, either for performance reasons or lack of hardware support.
+Other than with cast operations offering an explicitly marked unsafe operator mathematical operator is more 
```

Comment:
this sentence is not clear to me 😢

Reply:
Changed it to "Having different operators for safe floating point operations and unsafe integer operations would force users to
use different symbols for quantities using floating-point or integral values which decreases usability significantly. " hope it is now somewhat clearer what I wanted to say